pkg/distributor/writefailures/manager.go

```diff
@@ -32,6 +32,10 @@ func NewManager(logger log.Logger, cfg Cfg, tenants *runtime.TenantConfigs) *Man
 }
 
 func (m *Manager) Log(tenantID string, err error) {
+	if m == nil {
```

Comment:
this is useful for tests so that you don't have to assign a valid manager instance to every ingester/distributor